distribution/lib/Standard/Base/0.0.0-dev/src/Data/Filter_Condition.enso

```diff
@@ -22,25 +23,25 @@ polyglot java import org.enso.base.Regex_Utils
 
 type Filter_Condition
     ## Is less than a value (or another column, in case of Table operations)?
-    Less than:Any action:Filter_Action=Filter_Action.Keep
+    Less than:Any=(Missing_Argument.throw "than") action:Filter_Action=Filter_Action.Keep
```

Comment:
Why use `throw` over `ensure_present`?

Reply:
The source location was not being displayed anyway.

IMHO `ensure_present` has the nice advantage that it shows which function this argument belongs to (unless that's broken?). Then it makes it easier to localize where the missing thing is in a big expression (e.g. a join with many parameters).

But OK to just keep simple for now if you think that's better, we can always change in the future.